x-pack/packages/kbn-elastic-assistant-common/impl/schemas/prompts/bulk_crud_prompts_route.schema.yaml

```diff
@@ -111,15 +118,21 @@ components:
         name:
           type: string
         promptType:
-          type: string
+          $ref: '#/components/schemas/PromptType'
         content:
           type: string
+        categories:
+          type: array
+          items:
+            type: string
+        color:
+          type: string
         isNewConversationDefault:
           type: boolean
         isDefault:
           type: boolean
-        isShared:
-          type: boolean
+        consumer:
```

Comment:
should this be enum?

Reply:
his value will come from the Kibana `applications.currentAppId$`, which is dynamic and we should be flexible enough to supporting new apps